src/app.rs

```diff
@@ -84,10 +90,33 @@ impl App {
                     DisplayMode::ASCII => DisplayMode::DEFAULT,
                     DisplayMode::DEFAULT => DisplayMode::ASCII,
                 };
+                self.update_config();
             }
             3 => self.show_help_popup = true,
             4 => self.current_page = Pages::Credit,
             _ => {}
         }
     }
+
+    pub fn update_config(&self) {
+        let config_path = dirs::home_dir()
+            .unwrap()
+            .join(".config/chess-tui/config.toml");
```

Comment:
here you could use the home dir like this 

```rust
        let home_dir = match dirs::home_dir() {
            Some(dir) => dir,
            None => {
                eprintln!("Impossible to get your home directory");
                return ();
            }
        };
```

But in the end this home dir code seems to be use so many times you could put it in the constant file to reimport it everywhere